src/daemon.rs

```diff
@@ -695,6 +772,11 @@ fn parse_config(config_file: &str) -> Result<Ini, String> {
         return Err("Missing global.role setting".to_string());
     }
 
+    #[cfg(feature = "kafka")]
```

Comment:
TODO: If have_kafka and sasl-password is not empty then ca-file must not be empty.